app/Http/Controllers/Backend/Access/User/UserController.php

```diff
@@ -63,7 +63,19 @@ public function create(ManageUserRequest $request)
      */
     public function store(StoreUserRequest $request)
     {
-        $this->users->create(['data' => $request->only('name', 'email', 'password', 'status', 'confirmed', 'confirmation_email'), 'roles' => $request->only('assignees_roles')]);
+        $this->users->create(
+            [
+                'data' => $request->only(
+                    'first_name',
+                    'last_name',
+                    'email',
+                    'password',
+                    'status',
+                    'confirmed',
+                    'confirmation_email'
```

Comment:
That's correct. Confirmation code should never be in the request as it's generated when it's needed. That one is this: 

<img width="395" alt="screen shot 2017-05-08 at 21 50 02" src="https://cloud.githubusercontent.com/assets/6026504/25832400/60e8a8e0-3438-11e7-8fdb-61ede0fa457b.png">